src/Core/ServiceCollectionExtensions.cs

```diff
@@ -33,8 +33,10 @@ public static IGraphQLBuilder AddGraphQL(this IServiceCollection services, Func<
 
             services.TryAddSingleton<IDocumentWriter>(x =>
             {
-                throw new InvalidOperationException("IDocumentWriter not set in DI. " +
-                    "Add a IDocumentWriter implementation, for example GraphQL.NewtonsoftJson.DocumentWriter or GraphQL.SystemTextJson.DocumentWriter");
+                throw new InvalidOperationException(
+                    "IDocumentWriter not set in DI. " +
+                    "Add a IDocumentWriter implementation, for example " +
+                    "GraphQL.SystemTextJson.DocumentWriter or GraphQL.NewtonsoftJson.DocumentWriter.");
```

Comment:
Also I think it could be right to give a link to README.md here: For more information see https://github.com/graphql-dotnet/graphql-dotnet/blob/master/README.md